andela/ah-frontend-targaryen
# #161776758 delete articles

#### What does this PR do?

Allow authenticated users to delete articles they have authored
#### Description of Task to be completed?

Currently, our application has no capability of deleting articles

This PR allows authenticated users to delete articles they have authored


#### How should this be manually tested?

- Run the application `npm start`
- Click `Read More` on one of the articles you have authored
- Click the `delete` button and confirm the delete

#### What are the relevant pivotal tracker stories?

[#161776776](https://www.pivotaltracker.com/story/show/161776776)



#### Any background context you want to add?
N/A



#### Screenshots
![image](https://user-images.githubusercontent.com/9901011/48762623-796d1980-ecbc-11e8-8fbc-734604bdf51e.png)

## Checklist: I completed these to help reviewers
